uiowa/uids
# Feature: Banner updates for heading sizes

This pull request adds a `headline--large` , `headline--medium`, `headline--small` sizes for banners to bypass the default heading scales provided by `headings.scss`.  

This pull request also updates the sizing for Zilla headings to be comparable to the default Roboto sizes. 
<details>
Before:

<img width="1078" alt="Screen Shot 2021-05-26 at 2 20 22 PM" src="https://user-images.githubusercontent.com/1036433/119718770-95257d80-be2d-11eb-9561-0c220cc1396a.png">

After:

<img width="1190" alt="Screen Shot 2021-05-26 at 2 19 59 PM" src="https://user-images.githubusercontent.com/1036433/119718778-96ef4100-be2d-11eb-9e19-db65eb05f254.png">

# Remaining tasks
- [x] Add small, medium, large sizes using clamp mixin to banners.scss
- [x] Remove legacy `bold-headline` classes from banners and slider
</details>

# How to test
- Verify the small, medium, large sizes are set on https://uids.brand.uiowa.edu/branches/feature_banner_heading_size_fixed/components/preview/banner--default.html, https://uids.brand.uiowa.edu/branches/feature_banner_heading_size_fixed/components/preview/banner--medium_centered_banner.html, https://uids.brand.uiowa.edu/branches/feature_banner_heading_size_fixed/components/preview/banner--small_centered_banner.html
- View the sizing updates to the `.headline-serif` heading on https://uids.brand.uiowa.edu/branches/feature_banner_heading_size_fixed/components/detail/paragraph.html.
- Compare slider heading sizes https://uids.brand.uiowa.edu/branches/feature_banner_heading_size_fixed/components/preview/slider--default.html vs https://uids.brand.uiowa.edu/components/preview/slider--default.html.  Should be similar.
